doc/Caliptra.md

```diff
@@ -1255,9 +1260,18 @@ The Caliptra subsystem offers a complete RoT subsystem, with open source program
 *Figure: Caliptra security subsystem*
 ![](./images/Subsystem.png)
 
-**Caliptra Subsystem Architectural Flows**
+# Caliptra Subsystem Trademark Compliance 
+- Caliptra subsystem trademark compliance shall have Caliptra core 2.0, Life Cycle Controller, Fuse Controller, I3C with recovery interface, MCU and MCI taken as is without change to ensure there is hardware transparency and consistency. 
+- Caliptra subsystem provides life cycle state to the SOC.
```

Comment:
Didnt understand but we are being prescriptive in terms of RTL to take on what can and cannot be changed.

Reply:
Any update on this comment @bluegate010 ?

Reply:
Ah sorry, I missed your first reply earlier.

I don't have an issue with the content, but it seems like this describes something that's true about Subsystem, rather than something specific to trademark compliance. Another way of putting it: there are many aspects of Subsystem that we would not want integrators to change and still be able to achieve trademark compliance. Why single the lifecycle state out in particular?

Perhaps this line could go up above within an existing paragraph that describes Subsystem's properties. Something like:

"Fundamental difference between passive mode and subsystem mode is that in the subsystem mode Caliptra is the RoT for the SoC and provides lifecycle state, streaming boot, secure boot, and attestation."